scripts/setup/config.ps1

```diff
@@ -45,13 +45,38 @@ Param (
     
     [parameter()]
     [string]
-    $Destination
+    $Destination,
+
+    [parameter()]
+    [string]
+    $IISAdminOwnersGroup = "IIS Administration API Owners",
```

Comment:
Looks like this should be in [Globals.ps1](https://github.com/Microsoft/IIS.Administration/blob/dev/scripts/setup/globals.ps1)

Reply:
Moving